### PR TITLE
TST: tests using invalid_scalar fixture

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
https://github.com/pandas-dev/pandas/runs/4051113097?check_suite_focus=true looks to be an actual issue

**Review by jreback — APPROVED** [review]

File: pandas/tests/extension/base/getitem.py [hunk 1]
```diff
@@ -120,6 +120,33 @@ def test_getitem_scalar(self, data):
...
+            data[2.5]
+
+        ub = len(data)
+        msg = "|".join(
+            [
+                "list index out of range",  # json
+                "index out of bounds",  # pyarrow
+                "Out of bounds access",  # Sparse
+                f"index {ub+1} is out of bounds for axis 0 with size {ub}",
+                f"index -{ub+1} is out of bounds for axis 0 with size {ub}",
```
Comment (jreback) [inline pandas/tests/extension/base/getitem.py:142]: > >       with pytest.raises(IndexError, match=msg):
E       AssertionError: Regex pattern 'list index out of range|index out of bounds|Out of bounds access|index 101 is out of bounds for axis 0 with size 100|index -101 is out of bounds for axis 0 with size 100' does not match 'loc must be an integer between -100 and 100'.

i think these are asserting on 3.10 builds

Comment (jreback) [reply pandas/tests/extension/base/getitem.py:142]: https://github.com/pandas-dev/pandas/runs/4058191626?check_suite_focus=true

Comment (jbrockmendel) [reply pandas/tests/extension/base/getitem.py:142]: should be addressed by just-pushed commit in #44242

**Pull request merged**